hail/src/main/scala/is/hail/backend/service/Worker.scala

```diff
@@ -100,7 +99,7 @@ object Worker {
     timer.start("executeFunction")
 
     val hailContext = HailContext(
-      ServiceBackend(), skipLoggingConfiguration = true, quiet = true)
+      new ServiceBackend(null), skipLoggingConfiguration = true, quiet = true)
```

Comment:
Could you add a comment or fixme as to why there's this dummy backend on the worker? I realize it might not be easily fixable (I think you tried on an earlier revision) but it's odd-looking

Reply:
fixme added. I forget exactly why workers need a HailContext, but to have a HailContext one must have a backend.